### ISSUE TITLE
Time based tracking

### ISSUE THREAD

**Issue by se1exin** [nehero/expense-tracker#13]
Just putting an idea out there, but what do you think of making the tracker time based? E.g. you enter the frequency of your incomes and expenditures, and it tells you whats incomes/expenses are coming up.

I imagine it as the app having two views (e.g. two tabs) - the default one that tells you your incomes/expenses from a top level - like how it currently does, and a second that shows the incomes/expenses in a time based sense.

Thoughts?

**Comment by nehero** [comment]
That's a really good idea, which would make reoccurring expenses get added automatically. I'm thinking you might need a server for them to actually get applied at the desired time though.

**Comment by se1exin** [comment]
I don't think new expenses would be need to added automatically, instead of having a system that automatically adds to the array of expenses (causing duplication), you would have the system calculate and display past/future expenses based off a source expense's start date and frequency. As long as you have a reference starting date and a frequency you will be able to calculate every historic event for a given expense relative to the system's current datetime. Does that make sense?

In other words, the first view would simply be a visual representation of the Income and Expense arrays. The second view would be a dynamically calculated representation of each array with each item being shown as the last in/out and/or next in/out for that income/expense as a function of start date * frequency, relative the current time on the device

**Comment by nehero** [comment]
Ohh I gotcha. We should work in a time stamp for all the current values entered in too. I've added you as a collaborator to this repo so you don't have to submit pull requests anymore.

**Comment by se1exin** [comment]
Closing to clean up old still-open issues
